packages/browser/test/unit/tracekit/chromium.test.ts

```diff
@@ -430,4 +430,71 @@ describe('Tracekit - Chrome Tests', () => {
       },
     });
   });
+
+  it.only('should parse exceptions with frames without full paths', () => {
```

Comment:
```suggestion
  it('should parse exceptions with frames without full paths', () => {
```

Reply:
I would say that there should be a lint for that but I regularly push changes without checked linting anyway!